.github/workflows/build.yml

```diff
@@ -59,6 +59,15 @@ jobs:
       - name: Linux Before install
         run: sudo ./.travis/${{ matrix.os_name }}_runner.sh before_install
 
+      - name: Start Dubbo Backend
+        if: matrix.os_name == 'linux_openresty'
+        run: |
+          sudo apt install -y maven
+          cd t/lib/dubbo-backend
+          mvn package
+          cd dubbo-backend-provider/target
+          java -Djava.net.preferIPv4Stack=true -jar dubbo-demo-provider.one-jar.jar > /tmp/java.log &
```

Comment:
LGTM

It is better to move `t/lib/dubbo-backend` to a separate repo, and only need to download the compiled jar when running CI.

Reply:
As the content of Java code is depended on what you want to test, it is inconvenient to move it to a separate repo.

Reply:
`mvn package` + `java -Djava.net.preferIPv4Stack=true`

How long does it usually take?

Reply:
@membphis 
32 seconds

Reply:
I think this is acceptable